test/unit/test.discovery.v1.js

```diff
@@ -47,6 +47,8 @@ describe('discovery-v1', function() {
       .reply(200, { environment_id: 'yes' })
       .get(paths.environmentinfo + '?version=' + service.version_date)
       .reply(200, { environment_id: 'info' })
+      .put(paths.environmentinfo + '?version=' + service.version_date)
```

Comment:
Ideally I'd like to see these added to the test where they're used rather than a giant list at the top, but not a blocker since you're following the existing pattern.